src/index.ts

```diff
@@ -102,6 +102,29 @@ export interface BigtableOptions extends gax.GoogleAuthOptions {
   BigtableTableAdminClient?: gax.ClientOptions;
 }
 
+/**
+ * Retrieves the domain to be used for the service path.
+ *
+ * This function retrieves the domain from gax.ClientOptions passed in or via an environment variable.
+ * It defaults to 'googleapis.com' if none has been set.
+ * @param {gax.ClientOptions} [opts] The gax client options
+ * @returns {string} The universe domain.
+ */
+function getDomain(opts?: gax.ClientOptions) {
+  // This code for universe domain was taken from the Gapic Layer.
+  // It is reused here to build the service path.
+  const universeDomainEnvVar =
+    typeof process === 'object' && typeof process.env === 'object'
+      ? process.env['GOOGLE_CLOUD_UNIVERSE_DOMAIN']
+      : undefined;
+  return (
+    opts?.universeDomain ??
+    opts?.universe_domain ??
```

Comment:
Is this a nodeJS convention to support both casings? AFAIK this is not the case in any other language implementation

Reply:
This is code is taken from [here](https://github.com/googleapis/nodejs-bigtable/blob/589540475b0b2a055018a1cb6e475800fdd46a37/src/v2/bigtable_client.ts#L120-L129). I suggest we leave it as is because this is just Gapic code, but if you have an exact suggestion then maybe I could try that as well?

Reply:
In the JSON file, it's defined as `universe_domain`. I don't have a exact suggestion other than to say since the language agnostic spec doesn't require both casings, it must be specific to NodeJS.

According to https://github.com/googleapis/nodejs-bigtable/pull/1386, that's how support was added for all GAPICs, so LGTM

Reply:
Ok. Thanks for looking into that.